src/components/Dashboard/RewardsModule.js

```diff
@@ -31,50 +33,72 @@ const useTotalDisputesFees = (arbitrableFees, appealFees) => {
 // Dispute fees + Appeal Fees => DAI fees
 const RewardsModule = React.memo(function RewardsModule({
   rewards,
+  treasury,
   loading,
+  onWithdraw,
   onSettleReward,
   onSettleAppealDeposit,
 }) {
   const wallet = useWallet()
+  const { feeToken } = useCourtConfig()
+
+  const { totalArbitrableFees, totalAppealFees } = useTotalDisputesFees(
+    rewards?.arbitrableFees,
+    rewards?.appealFees
+  )
+
+  // TODO: Handle possible multiple tokens (fee token can change)
+  const treasuryToken = treasury?.find(({ token }) =>
+    addressesEqual(token.id, feeToken.id)
+  )
+  const treasuryBalance = treasuryToken ? treasuryToken.balance : bigNum(0)
+  const totalDisputesFees = totalArbitrableFees.add(totalAppealFees)
+  const totalFees = totalDisputesFees.add(treasuryBalance)
 
   // Form submission
   const handleFormSubmit = async event => {
```

Comment:
This should probably be in a `useCallback()`

Reply:
Now that we are not passing the function via props i think that is not needed any more ?

Reply:
It seems like `handleFormSubmit()` is still being passed to the `form`'s `onsubmit` prop.

But usually, the rule of thumb:
- If the function is dependent on component state / props, probably best to wrap it in a `useCallback()` so it doesn't get re-created on each render
- Otherwise, it could likely be moved to be outside of the component